conformity/fields/basic.py

```diff
@@ -26,29 +26,43 @@ def introspect(self):
         raise NotImplementedError("You must override introspect() in a subclass")
 
 
-@attr.s
 class Constant(Base):
     """
-    Value that must match exactly.
+    Value that must match exactly. You can pass a series of options
+    and any will be accepted.
     """
 
-    value = attr.ib()
-    description = attr.ib(default=None)
+    def __init__(self, *args, **kwargs):
+        self.values = args
+        if not self.values:
+            raise TypeError("You must provide at least one constant value")
+        self.description = kwargs.get("description", None)
+        # Check they didn't pass any other kwargs
+        if list(kwargs.keys()) not in ([], ["description"]):
+            raise TypeError("Invalid keyword arguments for Constant: %s" % kwargs.keys())
 
     def errors(self, value):
         """
         Returns a list of errors with the value. An empty/None return means
         that it's valid.
         """
-        if value != self.value:
-            return [
-                Error("Value is not %r" % self.value),
-            ]
+        if value not in self.values:
+            if len(self.values) == 1:
+                return [
+                    Error("Value is not %r" % (self.values[0], )),
+                ]
+            else:
+                return [
+                    Error("Value is not one of: %s" % (
```

Comment:
What's the default error `code` when all you pass to `Error` is a `message`? `INVALID`? In old SOA, we used `UNKNOWN` for when they passed a value that didn't match the enum set. I wonder if that would make more sense than `INVALID`.